lib/stytch.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
-require 'faraday'
-require 'faraday_middleware'
+require "faraday"
+require "faraday_middleware"
 
-require_relative 'stytch/client'
-require_relative 'stytch/middleware'
-require_relative 'stytch/version'
+require_relative "stytch/b2b_client"
+require_relative "stytch/client"
+require_relative "stytch/middleware"
+require_relative "stytch/version"
 
 module Stytch end
+module StytchB2B end
```

Comment:
auto-formatter preferred double quotes

Other question -- do I have to declare StytchB2B like this?